webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    index: './src/index.js',
+  },
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+    publicPath: '/',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+  optimization: {
+    runtimeChunk: 'single',
+  },
+};
```

Comment:
- [ ] When I start the server on port 8080, I get this error message:

![image](https://user-images.githubusercontent.com/77106962/212994680-4724f1a6-6ac4-41bb-b894-84269d7d0ffb.png)

You can resolve this by implementing the solution in this [article](https://webpack.js.org/configuration/mode/) 👍

Reply:
Bless you for this article @Gambit142 . Great help and review